tools/zbfind

```diff
@@ -930,7 +929,22 @@ class ZBPoller(Thread):
                 self.startinjtime = time.time()
                 self.kb.sniffer_on()
 
-            packetlist = self.kb.pnext()
+            try:
+                packetlist = self.kb.pnext()
+                if packetlist == None:
+                    thresh += 1
+                else:
+                    thresh = 0
+                if thresh > 25:  # thresh helps handle a bug in killerbee firmware for rzusbstick which makes the stick need to be reset.
```

Comment:
Is this something that was pretty constant after 25 messages that issues occurred? I haven't seen this, so wondering if it's something that is worth a dive into the firmware to look for. The RZUSBSTICK firmware was made from an Atmel source with only a few lines changed by Josh.